wpiutil/src/main/native/cpp/llvm/raw_ostream.cpp

```diff
@@ -580,28 +592,45 @@ void raw_fd_ostream::anchor() {}
 //  outs(), errs(), nulls()
 //===----------------------------------------------------------------------===//
 
-/// outs() - This returns a reference to a raw_ostream for standard output.
-/// Use it like: outs() << "foo" << "bar";
-raw_ostream &wpi::outs() {
+raw_fd_ostream &wpi::outs() {
   // Set buffer settings to model stdout behavior.
   std::error_code EC;
-  static raw_fd_ostream* S = new raw_fd_ostream("-", EC, fs::F_None);
+  static raw_fd_ostream S("-", EC, fs::OF_None);
```

Comment:
This reverts #1401.  We want to maintain this difference to upstream (for both outs and errs).

Reply:
👍